### ISSUE TITLE
about code

### ISSUE THREAD

**Issue by gangsterless** [junpan19/VSR_TGA#6]
As mentioned in [issue4](https://github.com/junpan19/VSR_TGA/issues/4) , the pretrained model cannot match the arch. I revised the arch according to the pretrained weights and made it run.  I just wanted to test the pretrained model with my own data,but  I got disordered result. like this:

![res](https://user-images.githubusercontent.com/43644055/109589351-2940bb00-7b45-11eb-80a2-12264f6f71a2.png)

  I also attempted to train from scratch, but loss was very big and decreased slowly, and after 30 epochs, I got similar disordered result like your pretrained model. I checked my data organization several times and I am really wonder what is the matter.
Your reply will be fully appreciated！！！
Thank you in advance!
